src/main/scala/uk/gov/nationalarchives/pentaho/helpers/ResultSetIterator.scala

```diff
@@ -0,0 +1,8 @@
+package uk.gov.nationalarchives.pentaho.helpers
```

Comment:
Can we change the base package to `uk.gov.nationalarchives.pdi` throughout please? This then matches what we have used in our Kettle plugins too.